create3_control/controllers/fb_linearization.py

```diff
@@ -20,45 +36,42 @@ def __init__(self, gain, length):
 
     def setup_goal(self, goal_pose, current_pose):
         self.relative_goal_pose = goal_pose
+        relative_goal_position = self.relative_goal_pose.position
 
-        ##This is commented out because projecting the goal forward of the reference lenght produces a worst result
-        # front_bumper_absolute_pose = utils.add_relative_to_absolute_pose(self.front_bumper_position, current_pose)
-        # self.absolute_goal_pose = utils.add_relative_to_absolute_pose(self.relative_goal_pose.position, front_bumper_absolute_pose)
-
-        self.absolute_goal_pose = utils.add_relative_to_absolute_pose(self.relative_goal_pose.position, current_pose)
+        self.absolute_goal_pose = add_rel_to_abs_pose(relative_goal_position, current_pose)
 
         print(f"current pose: {current_pose}")
         print(f"absolute goal: {self.absolute_goal_pose}")
 
-
     def step_function(self, current_pose):
-        front_bumper_absolute_pose = utils.add_relative_to_absolute_pose(self.front_bumper_position, current_pose)
+        front_bumper_abs_pose = add_rel_to_abs_pose(self.front_bumper_position, current_pose)
 
         assert self.absolute_goal_pose
 
-        e_x = front_bumper_absolute_pose.position.x - self.absolute_goal_pose.position.x
-        e_y = front_bumper_absolute_pose.position.y - self.absolute_goal_pose.position.y
-        e_z = front_bumper_absolute_pose.position.z - self.absolute_goal_pose.position.z
+        e_x = front_bumper_abs_pose.position.x - self.absolute_goal_pose.position.x
+        e_y = front_bumper_abs_pose.position.y - self.absolute_goal_pose.position.y
+        # e_z = front_bumper_abs_pose.position.z - self.absolute_goal_pose.position.z  # never used
```

Comment:
if never used, then remove it, do not leave commented code that is not useful

Reply:
ok